pkg/k8s/network_policy.go

```diff
@@ -86,13 +73,24 @@ func parseNetworkPolicyPeer(namespace string, peer *networkingv1.NetworkPolicyPe
 			lsr.Key = policy.JoinPath(k8sConst.PodNamespaceMetaLabels, lsr.Key)
 			peer.NamespaceSelector.MatchExpressions[i] = lsr
 		}
-	} else {
-		// Neither PodSelector nor NamespaceSelector set.
-		return nil
+
+		selector := api.NewESFromK8sLabelSelector(labels.LabelSourceK8sKeyPrefix, labelSelector, peer.PodSelector)
+		retSel = &selector
+	} else if peer.PodSelector != nil {
```

Comment:
Does this reorder break older policies that accidentally worked? we now prefer the namespace selector over the pod selector. This might change which becomes the policy in cases where both were present, no? maybe we validate this elsewhere.